docs/conf.py

```diff
@@ -41,7 +41,7 @@ def get_version(*file_paths):
 
 VERSION = get_version("../openedx_authz", "__init__.py")
 # Configure Django for autodoc usage
-os.environ["DJANGO_SETTINGS_MODULE"] = "test_settings"
+os.environ["DJANGO_SETTINGS_MODULE"] = "openedx_authz.settings.test"
```

Comment:
Why did this change? This is possibly what's breaking the docs build

Reply:
We don't have a test_settings file instead we use settings/test.py which in theory should work